saleor/order/migrations/0130_save_order_token_in_relation_models.py

```diff
@@ -147,20 +147,9 @@ class Migration(migrations.Migration):
             ALTER TABLE order_order_gift_cards
             ALTER COLUMN order_token SET NOT NULL;
 
-            DO $$
-            DECLARE query text;
-            BEGIN
-                query := FORMAT(
-                    'alter table order_order_gift_cards drop constraint %s',
-                    (
-                        select constraint_name from information_schema.table_constraints
-                        WHERE table_name = 'order_order_gift_cards'
-                            and constraint_type = 'FOREIGN KEY'
-                            and constraint_name like '%order_order_id'
-                    )
-                );
-                EXECUTE query;
-            END $$;
+            ALTER TABLE order_order_gift_cards
```

Comment:
What shall we do with 3.2 environments which already have this migration applied?

Reply:
cc @NyanKiyoshi

Reply:
It's just a simplification of the original migration: nothing to be done about it